core/src/main/java/com/google/errorprone/refaster/UInstanceOf.java

```diff
@@ -32,9 +33,11 @@
 @AutoValue
 abstract class UInstanceOf extends UExpression implements InstanceOfTree {
   public static UInstanceOf create(UExpression expression, UTree<?> type) {
-    return new AutoValue_UInstanceOf(expression, type);
+    return new AutoValue_UInstanceOf(null, expression, type);
   }
 
+  public abstract JCPattern getPattern();
```

Comment:
This method is problematic, as `JCPattern` has only been introduced in JDK 14. It implements the interface `com.sun.source.tree.PatternTree`, which seems like a marker interface: http://hg.openjdk.java.net/jdk/jdk14/file/6c954123ee8d/src/jdk.compiler/share/classes/com/sun/source/tree/PatternTree.java

I am not sure how we can implement this interface on JDK 14 without breaking JDK 8/11.